File: cp/A_Anton_and_Polyhedrons.py
def solve():
    face_dict = {
        "Tetrahedron": 4,
        "Cube": 6,
        "Octahedron": 8,
        "Dodecahedron": 12,
        "Icosahedron": 20
    }

    n = int(input())
    ans = 0
    for _ in range(n):
        s = input()
        ans += face_dict.get(s)
    
    print(ans)

solve()

File: cp/A_Anton_and_Polyhedrons.java
import java.util.Scanner;

/**
 * A_Anton_and_Polyhedrons
 */
public class A_Anton_and_Polyhedrons {

    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            int n = sc.nextInt();
            int face = 0;
            for (int i = 0; i < n; i++) {
                String s = sc.next();
                switch (s) {
                    case "Tetrahedron" -> face += 4;
                    case "Cube" -> face += 6;
                    case "Octahedron" -> face += 8;
                    case "Dodecahedron" -> face += 12;
                    case "Icosahedron" -> face += 20;
                    default -> {
                    }
                }
            }
            System.out.println(face);
        }
    }
}